pkg/utils/secrets/basic_auth.go

```diff
@@ -37,25 +40,29 @@ const (
 	DataKeyUserName = "username"
 	// DataKeyPassword is the key in a secret data holding the password.
 	DataKeyPassword = "password"
+	// DataKeyPasswordHash is the key in a secret data holding the hash of the password.
+	DataKeyPasswordHash = "passwordHash"
```

Comment:
```suggestion
	DataKeyPasswordHash = "bcryptPasswordHash"
```

or something similar